pkgs/development/libraries/pipewire/wireplumber.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  nix-update-script,
+
+  # base build deps
+  meson,
+  pkg-config,
+  ninja,
+
+  # docs build deps
+  python3,
+  doxygen,
+  graphviz,
+
+  # GI build deps
+  gobject-introspection,
+
+  # runtime deps
+  glib,
+  systemd,
+  lua5_4,
+  pipewire,
+
+  # options
+  enableDocs ? true,
+  enableGI ? stdenv.hostPlatform == stdenv.buildPlatform
+}:
+let
+  mesonEnable = b: if b then "enabled" else "disabled";
```

Comment:
Maybe name this `mesonEnableFeature`, see the discussion in https://github.com/NixOS/nixpkgs/pull/153658#discussion_r779913430

Reply:
Maybe this can go into some common module at some point then?

Reply:
Renamed for now.